build.gradle

```diff
@@ -68,11 +79,7 @@ dependencies {
     simulationRelease wpi.sim.enableRelease()
 
     testImplementation 'org.junit.jupiter:junit-jupiter:5.10.1'
-    implementation('org.carlmontrobotics:lib199') {
-        version {
-            branch = '2024'
-        }
-    }
+     implementation "com.github.deepbluerobotics:lib199:2024-SNAPSHOT"

```

Comment:
"2024-SNAPSHOT", should this be a tag? Admittedly this is nitpicky though. Everything works for me, including the syntax highlighting.